src/__tests__/custom-rules/custom-rules.spec.ts

```diff
@@ -45,6 +53,26 @@ describe("custom-rules", () => {
 		).toBe(ERROR_MESSAGE);
 	});
 
+	it("should support withinDays condition for Yup date type", () => {
```

Comment:
should this test as conditional rendering instead of as a validation rule?

Reply:
- the tests here are not necessary because we have already tested the validation in `date-field.spec.ts`
- create a new test suite for withinDays conditional rendering in `json-to-schema.spec.ts`
- schema should contain `showIf` because we are testing conditional rendering